include/Operator/OneElectronOperator.hpp

```diff
@@ -29,7 +29,8 @@ class OneElectronOperator : public BaseOperator {
 
     // GETTERS
     Eigen::MatrixXd get_matrix_representation() const { return this->matrix; }
-    
+    double get(size_t p, size_t q) { return this->matrix(p,q); }
```

Comment:
We might (anticipating on abstracting the effective representation of the operators) also implement `operator()`. Can you make an issue out of this?

Reply:
I don't understand your request.